src/PrestaShopBundle/Form/Admin/Sell/Product/Options/VisibilityType.php

```diff
@@ -66,12 +67,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('visibility', ChoiceType::class, [
                 'label' => false,
                 'choices' => $this->productVisibilityChoiceProvider->getChoices(),
+                'choice_attr' => $this->productVisibilityChoiceProvider->getChoicesAttributes(),
                 'expanded' => true,
                 'required' => false,
                 // placeholder false is important to avoid empty option in radio select despite required being false
                 'placeholder' => false,
                 'column_breaker' => true,
                 'modify_all_shops' => true,
+                'help' => '', // should be set to enable help block
+                'help_attr' => ['class' => 'js-description'],
```

Comment:
```suggestion
                'help_attr' => ['class' => 'js-visibility-description'],
```

Reply:
I suggest a more specific class here, `description` is a bit too generic and could be common with other fields